webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceFormDrawer.vue

```diff
@@ -283,6 +304,20 @@ export default {
         this.setSpaceTemplateProperties();
       }
     },
+    'space.invitedMembers': function (items){
+      this.selectedSpacesWithExternals = [];
+      items.filter(item => item.providerId === 'space')
+        .forEach(space => {
+          this.isCheckingExternals = true;
```

Comment:
can you explain what is the aim of this variable ?
Is it necessary  ?

Reply:
I added it to display a loading gif while its' checking but i didn't add it :D

Reply:
ok, so we can remove the variable

Reply:
removed